functions/src/index.ts

```diff
@@ -238,3 +330,19 @@ async function getPinsNearby(
 
   return snapshot.docs;
 }
+
+function calcDistanceMiles(
```

Comment:
definitely fine for now, but when the new cost / reward algos get merged we should switch to use the same distance calculation function.

Reply:
Agreed. If this gets merged in tonight maybe you can rebase and combine them on #34?

Reply:
Will do!